src/StudentOrganizer.Infrastructure/Services/UserService.cs

```diff
@@ -99,5 +99,11 @@ public async Task LoginAsync(Guid id, string email, string password)
 			JwtDto token = _jwtHandler.CreateToken(user.Id, user.Role);
 			_memoryCache.Set(id, token);
 		}
+
+		public async Task DeleteUser(DeleteUser command)
+		{
+			_userRepository.Delete(command.UserId);
```

Comment:
Trzeba byłoby sprawdzić najpierw czy user nie jest ostatnim adminem w grupie, jeśli nie to dopiero usunąć konto, a jeśli tak to zwrócić mu wiadomość, że najpierw musi dodać kogoś z grupy do administracji lub usunąć grupę